__tests__/components/Comments/ContainerComment.test.js

```diff
@@ -198,7 +198,7 @@ describe('ContainerComment should run correctly', () => {
       display: 'flex',
       flexDirection: 'row',
       paddingBottom: 14,
-      borderLeftColor: colors.gray1,
+      borderLeftColor: COLORS.balance_gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors.gray1` to `COLORS.balance_gray` is a good move towards standardizing color usage across the application. However, it's important to ensure that `COLORS.balance_gray` and `colors.gray1` are indeed the same color. If they're not, this could lead to visual inconsistencies in your app.
```diff
-      borderLeftColor: colors.gray1,
+      borderLeftColor: COLORS.balance_gray,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->